### DISCUSSION TITLE
安装了茉莉花插件之后但无法在Zotero中找到设置的界面

### DISCUSSION THREAD

**Discussion by shanyaotangzz** [l0o0/jasminum#307] [Q&A]
明明已经安装好插件了，但是在Zotero的编辑-设
![1726822244149](https://github.com/user-attachments/assets/e6e83452-981a-4e32-b0bd-28a9973fe82e)
![1726822196236](https://github.com/user-attachments/assets/3e06687c-2749-4fe5-988d-d38a8d9938ed)
置中却找不到茉莉花设置界面，重新安装了几次都没用，卸载重装Zotero也没用，求解！

**Comment by july-2024** [comment]
+1 我也是奇了怪了

**Comment by jiaojiaodubai** [comment]
最新版已经修复这个问题了，请更新到 -17 及以上版本
